Makefile

```diff
@@ -1,5 +1,6 @@
 # TODO: this is a bit minimalistic isn't it?
 
+CC ?= $(CROSS_COMPILE)gcc
```

Comment:
No need to set CC, its set implicitly.

Reply:
I'm changing the currently hard-coded compiler to a variable (see the `bluebinder` rule). This exact same method is already used on [audiosystem-passthrough](https://github.com/mer-hybris/audiosystem-passthrough/blob/cd73fcc/Makefile#L25) so I don't see why we can't use it here as well to make cross-compilation possible without having to hack the Makefile locally.